website/docs/integrations/github-actions/atmos-terraform-apply.md

```diff
@@ -37,51 +37,62 @@ We recommend combining this action with the [`affected-stacks`](/integrations/gi
+The config should have the following structure:
 
 ```yaml
-  # .github/workflows/atmos-terraform-apply.yaml
-  name: "atmos-terraform-apply"
-
-  on:
-    workflow_dispatch:
-    pull_request:
-      types:
-        - closed
-      branches:
-        - main
-
-  # These permissions are required for GitHub to assume roles in AWS
-  permissions:
-    id-token: write
-    contents: read
-
-  jobs:
-    apply:
-      runs-on: ubuntu-latest
-      steps:
-        - name: Terraform Apply
-          uses: cloudposse/github-action-atmos-terraform-apply@v2
-          with:
-            component: "foobar"
-            stack: "plat-ue2-sandbox"
+# ./rootfs/usr/local/etc/atmos/atmos.yaml
+integrations:
+  github:
+    gitops:
+      terraform-version: 1.5.2
+      infracost-enabled: false
+      artifact-storage:
+        region: us-east-2
+        bucket: cptest-core-ue2-auto-gitops
+        table: cptest-core-ue2-auto-gitops-plan-storage
+        role: arn:aws:iam::xxxxxxxxxxxx:role/cptest-core-ue2-auto-gitops-gha
+      role:
+        plan: arn:aws:iam::yyyyyyyyyyyy:role/cptest-core-gbl-identity-gitops
+        apply: arn:aws:iam::yyyyyyyyyyyy:role/cptest-core-gbl-identity-gitops
+      matrix:
+        sort-by: .stack_slug
+        group-by: .stack_slug | split("-") | [.[0], .[2]] | join("-")
 ```
 
-with the following configuration as an example:
+> [!IMPORTANT]
+> **Please note!** This GitHub Action only works with `atmos >= 1.63.0`. If you are using `atmos < 1.63.0` please use [`v1` version](https://github.com/cloudposse/github-action-atmos-terraform-plan/tree/v1).
+
 
 ```yaml
-  # .github/config/atmos-gitops.yaml
-  atmos-config-path: ./rootfs/usr/local/etc/atmos/
-  atmos-version: 1.65.0
-  aws-region: us-east-2
-  enable-infracost: false
-  group-by: .stack_slug | split("-") | [.[0], .[2]] | join("-")
-  sort-by: .stack_slug
-  terraform-apply-role: arn:aws:iam::yyyyyyyyyyyy:role/cptest-core-gbl-identity-gitops
-  terraform-plan-role: arn:aws:iam::yyyyyyyyyyyy:role/cptest-core-gbl-identity-gitops
-  terraform-state-bucket: cptest-core-ue2-auto-gitops
-  terraform-state-role: arn:aws:iam::xxxxxxxxxxxx:role/cptest-core-ue2-auto-gitops-gha
-  terraform-state-table: cptest-core-ue2-auto-gitops
-  terraform-version: 1.65.0
+# .github/workflows/atmos-terraform-apply.yaml
+name: "atmos-terraform-apply"
+
+on:
+  workflow_dispatch:
+  pull_request:
+    types:
+      - closed
+    branches:
+      - main
+
+# These permissions are required for GitHub to assume roles in AWS
+permissions:
+  id-token: write
+  contents: read
+
+jobs:
+  apply:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Terraform Apply
+        uses: cloudposse/github-action-atmos-terraform-apply@v2
+        with:
+          component: "foobar"
+          stack: "plat-ue2-sandbox"
+          sha: ${{ github.sha }}          
+          atmos-config-path: ./rootfs/usr/local/etc/atmos/
+          atmos-version: 1.63.0
```

Comment:
Also, I believe we should consider moving this into the `atmos.yaml` for a few reasons:
- Ensure GitHub Actions and local execution use the same version
- Make it optional
- It's error prone and easily missed. [See this thread in SweetOps](https://sweetops.slack.com/archives/C031919U8A0/p1716566318764329?thread_ts=1716486754.381799&cid=C031919U8A0)

Reply:
If we support a version parameter, I think it doesn't belong in the integrations section, but more of a top-level section. Let's discuss with @aknysh

Reply:
@osterman 

We can not move into `atmos.yaml`

```yaml
atmos-config-path: ./rootfs/usr/local/etc/atmos/
atmos-version: 1.63.0
```

Because this setting is required by GitHub action to install Atmos and get gitops settings from `atmos.yml` config